website/src/components/canvas/CanvasElement.jsx

```diff
@@ -19,9 +21,10 @@ class CanvasElement extends React.Component {
    */
   constructor(props) {
     super(props);
-    this.elemRef;
+    this.elementRef;
```

Comment:
Just a reference to the element so that we can later control it with a function. Shouldn't be in the store because it is not actually a state, just a pointer to our object.